### PR TITLE
Add into main/Added more text

### PR DESCRIPTION
**Pull request by ROlearyPro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 wrertdgxyghhujkil
 
-awsetfrawerf
\ No newline at end of file
+awsetfrawerf
+
+THIS IS PLACEHOLDER TEXT, BLAH BLAH
```
Comment (CoryBretsch) [inline README.md:5]: I like what you did here! looks solid.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 wrertdgxyghhujkil
 
-awsetfrawerf
\ No newline at end of file
+awsetfrawerf
+
```
Comment (CoryBretsch) [inline README.md:4]: testing 123

**Review by CoryBretsch — CHANGES_REQUESTED** [review]
maybe i need to submit the review

File: README.md (hunk 2 above)
Comment (ROlearyPro) [reply README.md:4]: Tested, tested, 456

**Pull request merged**